From 943dd3c2b9d68385c9e71b1cc215cf03e2dd74fd Mon Sep 17 00:00:00 2001
From: "Tan, Raymond" <raymond.tan@intel.com>
Date: Wed, 29 Aug 2018 13:35:44 +0800
Subject: [PATCH 1/3] zlib-qat: add a install target to makefile

Upstream-Status: Inappropriate [Configuration]$

This adds an install target to qat_zlib_test makefiles
to facilitate the installation of test binaries to {D}.

Signed-off-by: Tan, Raymond <raymond.tan@intel.com>
---
 contrib/qat/qat_zlib_test/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/contrib/qat/qat_zlib_test/Makefile b/contrib/qat/qat_zlib_test/Makefile
index 7b86028..47829ac 100644
--- a/contrib/qat/qat_zlib_test/Makefile
+++ b/contrib/qat/qat_zlib_test/Makefile
@@ -132,5 +132,7 @@ comptestappsh$(EXE): $(OBJS) Makefile
 	$(CC) -o comptestappsh $(OBJS) $(COVERAGE) \
 	$(SHAREDLIBQAT)
 
+install:
+	cp comptestapp $(DESTDIR)$(bindir)
 clean:
 	rm -f $(OBJS) $(COVERAGE_OBJS) comptestapp comptestappsh
-- 
1.9.1

